matrix_client/room.py

```diff
@@ -477,23 +474,21 @@ def add_room_alias(self, room_alias):
     def get_joined_members(self):
         """Returns list of joined members (User objects)."""
         if self._members:
-            return self._members
+            return list(self._members.values())
```

Comment:
Can't think of it either.  
I don't think we should allow modifying the members of a room though, so we should keep `Room._members` and add `Room.members` as a property.  
Also it would be better to return a dict than a list, since it's easy for the caller to use `dict.values()` if they want a list, and I can see the check `user_id in Room.members` having its use.